build/tools/travis-ci.sh

```diff
@@ -129,7 +150,12 @@ case $wxTOOLSET in
Grid::ResizeScrolledHeader Grid::ColumnMinWidth Grid::AutoSizeColumn GridBlockCoords::Canonicalize GridBlockCoords::Intersects GridBlockCoords::Contains GridBlockCoords::Difference GridBlockCoords::SymDifference HeaderCtrlTestCase HtmlListBoxTestCase HyperlinkCtrlTestCase wxControl::Label wxControl::RemoveMnemonics wxControl::FindAccelIndex ListbookTestCase ListBoxTestCase ListCtrlTestCase ListViewTestCase MarkupTestCase NotebookTestCase wxNotebook::AddPageEvents OwnerDrawnComboBoxTestCase ColourPickerCtrlTestCase DirPickerCtrlTestCase FilePickerCtrlTestCase FontPickerCtrlTestCase RadioBoxTestCase RadioButton::Click RadioButton::Value RadioButton::Group RadioButton::Single RadioButton::Focus RearrangeListTestCase RichTextCtrlTestCase wxSearchCtrl::Focus wxSearchCtrl::ChangeValue SimplebookTestCase SliderTestCase SpinCtrlDoubleTestCase SpinCtrlDoubleTestCase::InitialDigits SpinCtrl::Init SpinCtrl::Init2 SpinCtrl::Init3 SpinCtrl::Init4 SpinCtrl::NoEventsInCtor SpinCtrl::Arrows SpinCtrl::Wrap SpinCtrl::Range SpinCtrl::Value SpinCtrl::Base SpinCtrl::SetValueInsideEventHandler TextCtrlTestCase wxTextCtrl::ProcessEnter wxTextCtrl::GetBestSize wxTextCtrl::LongPaste wxTextCtrl::EventsOnCreate wxTextCtrl::InitialCanUndo ToggleButtonTestCase ToolbookTestCase TreebookTestCase TreeCtrlTestCase TreeListCtrlTestCase VirtListCtrlTestCase Window::ShowHideEvent Window::KeyEvent Window::FocusEvent Window::Mouse Window::Properties Window::ToolTip Window::Help Window::Parent Window::Siblings Window::Children Window::Focus Window::Positioning Window::PositioningBeyondShortLimit Window::Show Window::Enable Window::FindWindowBy Window::SizerErrors ModalDialogsTestCase EventClone EvtloopTestCase EventPropagationTestCase KeyboardEventTestCase ExecTestCase wxExecute::RedirectUTF8 wxFont::Construct wxFont::Size wxFont::Style wxFont::Weight wxFont::GetSet wxFont::NativeFontInfo wxFont::NativeFontInfoUserDesc ImageTestCase wxImage::Paste ImageRawTestCase wxHtmlParser::ParseInvalid wxHtmlCell::Detach HtmlWindowTestCase wxHtmlPrintout::Pagination wxAcceleratorEntry::Create wxAcceleratorEntry::StringTests MenuTestCase wxMenuItemAccelEntry MiscGUIFuncsTestCase SelStoreTestCase GarbageTestCase SettingsTestCase SocketTestCase wxPersistTLW wxPersistDVC RowHeightCacheTestCase::TestRowRangesSimple RowHeightCacheTestCase::TestRowRangesGapsMod2 RowHeightCacheTestCase::TestRowRangesCleanUp1 RowHeightCacheTestCase::TestRowRangesCleanUp2 RowHeightCacheTestCase::TestHeightCache BoxSizer::Size1 BoxSizer::Size3 BoxSizer::CalcMin BoxSizer::SetMinSize BoxSizer::BestSizeRespectsMaxSize BoxSizer::RecalcSizesRespectsMaxSize1 BoxSizer::RecalcSizesRespectsMaxSize2 BoxSizer::IncompatibleFlags BoxSizer::Replace GridSizerTestCase wxWrapSizer::CalcMin wxWrapSizer::CalcMinFromMinor wxTopLevel::Show wxTopLevel::ShowEvent NumValidatorTestCase wxTextValidator::IsValid wxWindow::ClientWindowSizeRoundTrip wxWindow::MinClientSize wxWindow::SetSize wxWindow::GetBestSize wxWindow::MovePreservesSize XrcTestCase"
```

Comment:
OK, this definitely merits an explanation -- why do we need to run each test individually?

Also, even if this is really needed (and I hope we can avoid it), the list of tests must be retrieved from `./test_gui --list-test-names-only` output and not hardcoded like this, as this is completely unmaintainable.